apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -71,6 +71,17 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 			{ domain?.domain_name ? (
 				<p>
 					{ __( 'Custom domain', 'full-site-editing' ) }: { domain.domain_name }
+					{ planProduct?.billingPeriod === 'MONTHLY' && (
+						<>
+							<br />
+							<span className="nux-launch__summary-item__domain-price">
+								{
```

Comment:
Tiny suggestion:

```suggestion
								{ __( 'Domain Registration', 'full-site-editing' ) }:{ ' ' }
								{
```
Ignoring the existing broken translations, the string should be already translated if we look at:
https://translate.wordpress.com/projects/wpcom/-all-translations/42132/

<img width="1112" alt="Screenshot 2021-01-27 at 17 39 16" src="https://user-images.githubusercontent.com/14192054/106014945-a8fee400-60c6-11eb-93e9-a6dff060014b.png">

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/87ce4f05fcb138ef308f915abb3818c6ad28341c